examples/ogl_tutorials/tutorial03_matrices/src/maths.ads

```diff
@@ -0,0 +1,34 @@
+
+with Ada.Numerics;
+with Ada.Numerics.Generic_Elementary_Functions;
+
+with GL.Types;
+
+package Maths is
```

Comment:
> The subtypes are provided for consistency with types in a set of tutorials.
> If you prefer I can remove them.

I would say that if the subtypes only exist to have type names similar to C, they should be removed, since this is not what subtypes in Ada are for. The subroutine names in OpenGLAda also differ and rightfully so (to comply to the *Ada Quality and Style Guide*), so having different names for the vector and matrix types is not a problem.